app/utils/deep_link/index.ts

```diff
@@ -114,37 +120,66 @@ export async function handleDeepLink(deepLinkUrl: string, intlShape?: IntlShape,
     }
 }
 
+const matcherOpts: Parameters<typeof match>[1] = {decode: decodeURIComponent};
```

Comment:
What's the scenario where the `decode` is needed?

Reply:
removed